crates/rover-client/src/operations/subgraph/publish/publish_mutation.graphql

```diff
@@ -25,6 +25,8 @@ mutation SubgraphPublishMutation(
       }
       didUpdateGateway: updatedGateway
       serviceWasCreated: wasCreated
+      launchCliCopy
```

Comment:
I think we can likely remove this field from the API and only return the launch URL. The copy can live in Rover proper so future changes to the copy are easy to make and it's clear where the message is coming from.

Reply:
This was a decision we came to after discussing with the houston and mercury teams. Having the copy in the client is cleaner but providing it from the API gives more flexibility as we change what we want to be displayed when a subgraph is published. For example, one addition planned in the near future is providing the user a list of contract launches in the publish results. If we pass this through `launchCliCopy` it can be changed on the fly without having to wait for rover releases or for clients to update their versions.